packages/bot-skeleton/src/scratch/hooks/tooltip.js

```diff
@@ -0,0 +1,64 @@
+import { blueInfo } from '../blocks/images';
+
+Blockly.Tooltip.HOVER_MS = 50;
+Blockly.Tooltip.LIMIT = 100;
+
+Blockly.Tooltip.show_ = function() {
+    let param;
+    Blockly.Tooltip.DIV = ((Blockly.Tooltip.poisonedElement_ = Blockly.Tooltip.element_), Blockly.Tooltip.DIV);
+    if (!Blockly.Tooltip.blocked_) {
+        Blockly.Tooltip.DIV.innerHTML = '';
+        for (param = Blockly.Tooltip.element_.tooltip; typeof param === 'function'; ) param = param();
+        param = Blockly.utils.string.wrap(param, Blockly.Tooltip.LIMIT);
+        param = param.split('\n');
+        for (let counter = 0; counter < param.length; counter++) {
```

Comment:
Could we use `forEach` instead of `for`? It's more functional and as you don't use `counter`, I think it's better.